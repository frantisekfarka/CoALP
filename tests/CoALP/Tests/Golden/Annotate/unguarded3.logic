stream2(scons(X,Y)):-bit(X), stream2(scons(X,Y)).
