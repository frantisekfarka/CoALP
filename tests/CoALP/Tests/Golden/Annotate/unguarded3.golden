stream2(scons(V_2i, V_3i), transform-func-1(V_4i, V_5c)) :- bit(V_2i, V_4i), stream2(scons(V_2i, V_3i), V_5c).
